src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,51 @@
+@import "../utils/variable";
+@import "../utils/mixins";
+
+.container {
+  @include square ($size-of-frame);
+  background-color: antiquewhite;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  position: relative;
+
+  &__minute {
+    height: 0.3 * $size-of-frame;
+    width: 0.03 * $size-of-frame;
+    background-color: rgb(224, 46, 46);
+    border-radius: 0.03 * $size-of-frame;
+
+    animation: move 3600s steps(60, end) infinite forwards;
```

Comment:
minutes hand usually moves smooth, not 1 step in a minute, use `linear` value for animation-timing-funtion